examples/simple_demo_qml/IgnitionRenderer.cc

```diff
@@ -178,6 +178,7 @@ ignition::rendering::CameraPtr CreateCamera(const std::string &_engineName)
   // ensure that the QML application and Ignition / Ogre2 share an OpenGL
   // context
   params["useCurrentGLContext"] = "1";
+  // params["metal"] = "1";
```

Comment:
remove?

Reply:
Yes, this PR shouldn't change anything in `simple_demo_qml`, there'll be a follow up PR for that.

Reply:
Fixed in 25d8719d